src/features/toast/Toast.tsx

```diff
@@ -2,15 +2,27 @@ import React from 'react'
 import classNames from 'classnames'
 import RunStatusIcon from '../run/RunStatusIcon'
 import IconButton from '../../chrome/IconButton'
+import { Link } from "react-router-dom";
+import { useSelector } from "react-redux";
+import { selectVersionInfo } from "../collection/state/collectionState";
+import { VersionInfo } from "../../qri/versionInfo";
 
 interface ToastProps {
   message: string
+  initID:string
   type: 'running' | 'succeeded' | 'failed'
-  id: string
 }
 
+const getRef = (vi: VersionInfo | undefined) => {
```

Comment:
nice, we already have something like this:
https://github.com/qri-io/frontend/blob/6726bc5c4f653ee66eb7dd3f362fa640e46115c3/src/qri/versionInfo.ts#L94-L96

We should modify it so that if there is no username or name then we return an empty string (rather than '/')